test/src/messaging/utils/config.spec.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    setAuthConfig,
+    getAuthConfig,
+    setAuthToken,
+    getAuthToken,
+    setAuthTokenRetryCount,
+    getAuthTokenRetryCount,
+} from '../../../../src/messaging/utils';
+
+const AUTH_CONFIG = require('../credentials.json');
+
+describe('config', () => {
```

Comment:
maybe a test for some of the exception scenario flows

Reply:
added test

Reply:
added more rainyday tests

```PASS  test/src/messaging/utils/config.spec.js
  config
    when setAuthConfig is called
      ✓ should return true given a payload (3ms)
      ✓ should return false given no payload
    when getAuthConfig is called
      ✓ should be true
      ✓ should return expected payload
    when setAuthToken is called
      ✓ should return true (1ms)
      ✓ should return false given no payload
    when getAuthToken is called
      ✓ should return a string
    when getAuthTokenRetryCount is called
      ✓ should return 0 as a number (1ms)
      ✓ should return 1 as a number
    when setAuthTokenRetryCount is called
      ✓ should get auth token retry count of 0 when storage not yet initialised
      ✓ should return true given a number (1ms)
      ✓ should return false given no payload```